packages/engineer/src/application-proxy-service.ts

```diff
@@ -0,0 +1,51 @@
+import { Application, IFeatureDefinition, IConfigDefinition, TopLevelConfigProvider } from '@wixc3/engine-scripts/src';
+import type { SetMultiMap, TopLevelConfig } from '@wixc3/engine-core/src';
+
+export class ApplicationProxyService extends Application {
```

Comment:
This class should stay, but it shouldn't extend application, all of the reused application methods should be extracted to utils and and somehow shared between engineer and scripts, but where should they live? @RomanYarik. Anyway I think we can live with it until after merge and start moving gradually in next PR's

Reply:
I am accessing protected members of application, I want to eventually extract those methods into a utils folder somewhere(not really sure where yet) but I need to extend it for now